web/templates/pages/about.html

```diff
@@ -17,10 +17,21 @@
 				<p class="intro">CodeCatz is a coding study group where women are always in majority. The group is dedicated to all women who want to learn coding, however men are welcomed too. Catz pass their coding knowledge to kittens through various group and/or personal projects and when the kittens are more skilled, they pass their gained knowledge to new beginners. </p>
 				<p><strong>The objective of CodeCats is to assist women to enter in to the world of coding and change the world</strong>.</p>
 				<p class="intro">CodeCatz have only <strong>two rules</strong> for the members: <strong>chocolate</strong> at the meetings and <strong>full dedication to coding</strong>.</p>
-				<p class="intro">We are always open for a new potential members, so come over to say hi, see how we work. Especially welcome if you're a female kitten, who wants to learn more about programming or an expert cat, who would like to help said kittens and learn new things from other Catz.</p>
+				<p class="intro">We are always open for a new potential members, so come over to say hi, see how we work. Especially welcome if you're a female kitten, who wants to learn more about programming or an expert cat, who would like to help kittens and learn new things from other Catz.</p>
 				<p class="intro">CodeCatz welcome also mentors or companies who see future of women in programming and would like to assist our kittens in any shape or form. </p>
 				<p>We code together <strong>every Wednesday</strong> at <strong>RAMPA Laboratorij, Kersnikova ulica 4, Ljubljana at 17:30</strong>.</p>
 			</div>
 		</div>
-
+		<div class="row">
+			<div class="col-lg-6 col-sm-5">
+				<div class="clearfix">
+					<a href="https://www.google.si/maps/place/Kersnikova+ulica+4/@46.0558232,14.5040802,17z/data=!4m3!3m2!1s0x4765329e4492e579:0x228cf6142750beae!4b1" target="_blank">
+					<iframe frameborder="0" style="border:0"src="https://www.google.com/maps/embed/v1/place?key=AIzaSyDeeTTMCKwoF-Np6-hzX2JuTiTsEk-SDYA
```

Comment:
space after src="", iframe put src=" .. " into one line, otherwise it might break in strange ways on different browsers